src/flake8_requirements/modules.py

```diff
@@ -496,9 +496,11 @@
 # Mapping for known 3rd party projects, which provide more than one module
 # or the name of the module is different than the project name itself.
 KNOWN_3RD_PARTIES = {
+    "allure-pytest": ["allure"],
```

Comment:
For databricks-connect and pyhamcrest I'm OK, but allure-pytest seems to be deprecated... https://github.com/allure-framework/allure-pytest . Maybe allure-python should be added instead?

Reply:
Good call, but from what I can tell the PyPI project is still https://pypi.org/project/allure-pytest/ - they just moved the repo to `allure-python`.

Reply:
I've just looked at this allure thingy, and things are more complicated (if one wants to do things right). The allure-python repository indeed contains allure-pytest package, but this package does not provide `allure` module. The `allure` module is provided by allure-python-common, which creates "namespace wrapper", hence it provides `allure` module.... 

I will merge it, but I will add small comment that "allure" is not directly provided by allure-pytest :D